crates/router/src/connector/trustpay/transformers.rs

```diff
@@ -1637,16 +1647,19 @@ pub struct Errors {
 }
 
 #[derive(Default, Debug, Serialize, Deserialize, PartialEq)]
+#[serde(rename_all = "camelCase")]
 pub struct TrustpayErrorResponse {
     pub status: i64,
     pub description: Option<String>,
     pub errors: Option<Vec<Errors>>,
+    pub instance_id: Option<String>,
 }
 
 #[derive(Deserialize)]
 pub struct TrustPayTransactionStatusErrorResponse {
     pub status: i64,
     pub payment_description: String,
+    pub instance_id: Option<String>,
```

Comment:
This struct is being used in PSync are we getting instance_id in PSync also?

Reply:
Yes we are getting instance_id in PSync response 

<img width="522" alt="Screenshot 2023-12-06 at 11 32 06 AM" src="https://github.com/juspay/hyperswitch/assets/131388445/695acaf7-93b9-4444-a2c8-1cfd19687b47">